docker-compose.yml

```diff
@@ -86,6 +86,29 @@ services:
       - --network-endpoint=https://polygon-rpc.com         # network endpoint, can choose your own endpoint
       - --token-duration=24                                # query auth token validity [hours]
       - --redis-endpoint=redis://indexer_cache
+      - --metrics-token=thisismyAuthtoken
+
+  # Uncomment to enable dashboard
+
+  # prometheus:
+  #   image: prom/prometheus:latest
+  #   container_name: indexer_prometheus_dev
+  #   volumes:
+  #     - ./metrics/prometheus.yml:/etc/prometheus/prometheus.yml
+  #   command: --config.file=/etc/prometheus/prometheus.yml
+  #   ports:
+  #     - 9090:9090
+
+  # grafana:
+  #   image: grafana/grafana:latest
+  #   container_name: indexer_grafana_dev
+  #   ports:
+  #     - 3000:3000
```

Comment:
Avoid to use `3000` on grafana, I remember there has other changes needed to make it works,  @nz-neo can provide help

Reply:
That's need change grafana's configure file, not only change ports. 
But I suggest we use default 3000 here, it reduce the difficulty to user, and also convenient for them to debug by themselves if need.

Reply:
Make sense